ci-operator/config/openshift/release/openshift-release-master__ocp-4.6.yaml

```diff
@@ -3,10 +3,18 @@ base_images:
     name: "4.6"
     namespace: ocp
     tag: ansible
+  assisted-installer:
+    name: "4.7"
```

Comment:
why I'm seeing 4.7 in configurations for ocp 4.6?

Reply:
if there's any reason behind that, I expect to see a comment explaining it

Reply:
Elaborate on the process on https://github.com/openshift/release/pull/15311.
Basically, because assisted aren't part of OCP, we don't have a fixed version of assisted for 4.6 nor 4.7.

But since all the other images already use 4.7 it was easier to make those changes with the same one without needing to import new images (Otherwise it would break current configurations. For example - etcd, mco, cvo, etc.)

Later on, I will move all of our images to a unique namespace.